mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -224,7 +227,13 @@ void onClickActivateClient() {
     }
 
     public void inflateClientInformation() {
-        mClientDetailsPresenter.loadClientDetailsAndClientAccounts(clientId);
+        ConnectivityManager connectivityManager = (ConnectivityManager) getContext().getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo networkInfo = connectivityManager.getActiveNetworkInfo();
+        if(networkInfo != null && networkInfo.isConnected()){
+            mClientDetailsPresenter.loadClientDetailsAndClientAccounts(clientId);
+            return;
+        }
+        showFetchingError("Device not connected to internet");
```

Comment:
Also try changing the hard-coded strings to string element

Reply:
I have followed the coding convention as in line 140 of [ClientDetailsPresenter.java](https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsPresenter.java) which uses the static string declaration while presenting `Client not found.` toast. 

Should I still change this??  Just need a confirmation on this. @robustTechie

Reply:
Try to change it that is good for long term

Reply:
@danishjamal104 it'd be great if you change it in that file as well.

Reply:
ok @dotrachit, ill make changes in both places.